pkgs/tools/archivers/unzip/default.nix

```diff
@@ -43,7 +43,12 @@ stdenv.mkDerivation rec {
       sha256 = "1jvs7dkdqs97qnsqc6hk088alhv8j4c638k65dbib9chh40jd7pf";
     })
     (fetchurl {
-      url = "https://sources.debian.org/data/main/u/unzip/6.0-26/debian/patches/06-initialize-the-symlink-flag.patch";
+      urls = [
+        # original link (dead):
+        "https://sources.debian.org/data/main/u/unzip/6.0-26/debian/patches/06-initialize-the-symlink-flag.patch"
```

Comment:
I think this might as well be a comment if it always 404's anyway?

Reply:
I made a choice to cater to some of a more security-concerned crowd at a cost of making an extra request.